root/etc/e-smith/events/actions/nethserver-mattermost-conf

```diff
@@ -20,21 +20,54 @@
 # along with NethServer.  If not, see COPYING.
 #
 
-su - postgres -c "scl enable rh-postgresql94 -- psql --port=55432 -lqt | cut -d \| -f 1 | grep -q -w mattermost"
+su - postgres -c "scl enable rh-postgresql12 -- psql --port=55434 -lqt | cut -d \| -f 1 | grep -q -w mattermost"
 if [ $? -eq 1 ]; then # database do not exists
+    su - postgres -c "scl enable rh-postgresql94 -- psql --port=55432 -lqt | cut -d \| -f 1 | grep -q -w mattermost"
+    if [ $? -eq 0 ]; then
+        echo "Migrating data from postgresql 9.4 to postgresql 12..."
```

Comment:
Since the action is executed inside an event, the output is logged in `/var/log/messages` and the output should be similar to other actions.
```suggestion
        echo "[NOTICE] Migrating data from postgresql 9.4 to postgresql 12"
```